src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -20,6 +20,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="AllocationHelper.cs" />
+    <Compile Include="Span\CovariantSliceNotSupported.cs" />
```

Comment:
nit: move this to the other itemgroup with the rest of the span tests here:
https://github.com/dotnet/corefx/pull/30411/files#diff-00a03683c006a09943017a6311ec51c9R36

Reply:
Actually remove this file all together.